### DISCUSSION TITLE
ヘッダーのロゴについて

### DISCUSSION THREAD

**Discussion by Fumiko-Nishiura** [next-l/enju_leaf#1713] [Q&A]
青翔中学・高等学校ボランティアの西浦です。お世話になっております。
ヘッダーのロゴを、学校のキャラクター画像と差替えたいのですが、
「更新する」を押下すると、以前のようにパスワードが更新されて
Enjuが使えなくなるかと心配です。
ご教示お願いします。

**Comment by nabeta** [comment]
@Fumiko-Nishiura この部分は、パスワードに関係しないので問題ありませんが、もし不具合がありましたらお知らせください。

**Comment by Fumiko-Nishiura** [comment]
田辺様
休暇中にもかかわりませず、誠にありがとうございました。
学校のキャラクターに変更できました。
ショートカットも同じキャラクターに変更することは可能でしょうか？
お時間のありますときにご教示ください。
西浦

2022年8月13日(土) 17:45 Kosuke Tanabe ***@***.***>:

> @Fumiko-Nishiura <https://github.com/Fumiko-Nishiura>
> この部分は、パスワードに関係しないので問題ありませんが、もし不具合がありましたらお知らせください。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/next-l/enju_leaf/discussions/1713#discussioncomment-3388778>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATNPS4QBS57DYDWF34Q3KF3VY5OBPANCNFSM56J7KMHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

─────────────────── ☆

西浦 富美子 / NISHIURA Fumiko

Email: ***@***.***

☆☆ ──────────────────

**Reply by nabeta** [reply]
すみません、ショートカットというのは画面のどの部分を指しますでしょうか。

**Comment by Fumiko-Nishiura** [comment]
申し訳ありません！ PCトップ画面におくアイコンのことを言おうとしていました。
アイコンを「青翔中学校・高等学校図書館」として添付の画像を入れたいです。
西浦

2022年8月20日(土) 2:18 Kosuke Tanabe ***@***.***>:

> すみません、ショートカットというのは画面のどの部分を指しますでしょうか。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/next-l/enju_leaf/discussions/1713#discussioncomment-3433369>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATNPS4VMEUEJWTU5FNJ7SDDVZ66XBANCNFSM56J7KMHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

─────────────────── ☆

西浦 富美子 / NISHIURA Fumiko

Email: ***@***.***

☆☆ ──────────────────
